addon/components/bs-form.js

```diff
@@ -143,6 +144,11 @@ export default Ember.Component.extend({
   submitOnEnter: false,
 
   /**
+   * If set to true novalidate attribute is present on form element
+   */
+  novalidate: false,
```

Comment:
- [ ] Please add the yuidoc tags @property, @type, @default and @public for correct API docs generation. 

Apart from that it looks good, thanks!